.env

```diff
@@ -1 +1,5 @@
-REACT_APP_GOOGLE_MAPS_API_KEY = ''
\ No newline at end of file
+REACT_APP_GOOGLE_MAPS_API_KEY = ''
+REACT_APP_BUYER_APP_ID=''
```

Comment:
oh this should not be here. this is in BAP

Reply:
updated